synapse/lib/stormtypes.py

```diff
@@ -4064,30 +4064,29 @@ async def cacheset(self, name, valu):
     @stormfunc(readonly=True)
     async def _getNodeData(self, name):
         name = await tostr(name)
-        confirm(('node', 'data', 'get', name))
```

Comment:
This is a documented permission for gating access to node data. This causes the ``test_storm_node_data`` test to fail.